core/README.md

```diff
@@ -1,23 +1,43 @@
-# Chainlink Core
+<br/>
+<p align="center">
+<a href="https://chain.link" target="_blank">
+<img src="../styleguide/static/images/logo-core-blue.svg" width="225" alt="Chainlink logo">
+</a>
+</p>
+<br/>
+
```

Comment:
I can add it back in. I had the Go badge in there originally, as well as others but I ended up removing them because technically the `core` directory isn't a package, the Go report cards reference the root project name, and I wanted to make sure this was noted beforehand if for some reason having icons in a place that technically "shouldnt" be there, but semantically can be justified is a cause for concern..